virttest/virt_vm.py

```diff
@@ -1216,6 +1216,9 @@ def print_guest_network_info():
                 if serial:
                     break
                 raise
+            except remote.LoginProcessTerminatedError as err:
+                error = err
+                break
```

Comment:
Is it better to use 'raise' instead of break? Because if 'break', then  print_guest_network_info() will be executed, but I do not think that is necessary. This exception is enough to be raised just like above LoginAuthenticationError. What do you think of? @xiaodwan

Reply:
Yes, 'raise' is better than 'break' here, thanks.